docs/software/sciapps/lammps.md

```diff
@@ -1,4 +1,338 @@
+#SBATCH --ntasks-per-core=1                                                    
+#SBATCH --ntasks-per-node=32 (2)
+#SBATCH --cpus-per-task=4 (3) 
+#SBATCH --account=<ACCOUNT> (4)
+#SBATCH --hint=nomultithread
+#SBATCH --hint=exclusive
+#SBATCH --constraint=mc                                                  
+#SBATCH --uenv=<LAMMPS_UENV>:/user-environment (5)
+#SBATCH --view=kokkos (6)
+
+ulimit -s unlimited
+
+export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK
+
+srun --cpu-bind=socket lmp -k on t $OMP_NUM_THREADS -sf kk -in lj_kokkos.in
+```
+
+1. Time format: `HH:MM:SS`.
+2. Number of MPI ranks per node.
+3. Number of threads per MPI rank.
+4. Change `<ACCOUNT>` to your project account name.
+5. Change `<LAMMPS_UENV>` to the name (or path) of the LAMMPS uenv you want to use.
+6. Enable the `kokkos` uenv view.
+
+Note that the same input file `lj_kokkos.in` can be used as with running LAMMPS with Kokkos on the HPC Platform.
+
+### Building LAMMPS from source
+
+#### Using CMake
+
+If you'd like to rebuild LAMMPS from source to add additional packages or to use your own customized code, you can use the develop views contained within the uenv image to provide you with all the necessary libraries and command-line tools you'll need.
+For the following, we'd recommend obtaining an interactive node and building inside the tmpfs directory.
+
+```bash
+salloc -N1 -t 60 -A <account>
+srun --pty bash
+mkdir /dev/shm/lammps_build; cd /dev/shm/lammps_build
+```
+
+After you've obtained a version of LAMMPS you'd like to build, extract it in the above temporary folder and create a build directory. 
+Load one of the two following views:
+
+=== "Kokkos"
+    ```bash
+    #build environment for lammps +kokkos package, without providing lmp executable
+    uenv start --view develop-kokkos lammps/2024:v2
+    ```
+=== "GPU"
+    ```bash
+    #build environment for lammps +gpu package, without providing lmp executable
+    uenv start --view develop-gpu lammps/2024:v2
+    ```
+
+and now you can build your local copy of LAMMPS. 
+For example to build with Kokkos and the `MOLECULE` package enabled:
+
+```
+CC=mpicc CXX=mpic++ cmake \
+-DCMAKE_CXX_FLAGS=-DCUDA_PROXY \
+-DBUILD_MPI=yes\
+-DBUILD_OMP=no \
+-DPKG_MOLECULE=yes \
+-DPKG_KOKKOS=yes \
+-DEXTERNAL_KOKKOS=yes \
+-DKokkos_ARCH_NATIVE=yes \
+-DKokkos_ARCH_HOPPER90=yes \
+-DKokkos_ARCH_PASCAL60=no \
+-DKokkos_ENABLE_CUDA=yes \
+-DKokkos_ENABLE_OPENMP=yes \
+-DCUDPP_OPT=no \
+-DCUDA_MPS_SUPPORT=yes \
+-DCUDA_ENABLE_MULTIARCH=no \
+../cmake  
+```
+
+!!! warning
+
+    If you are downloading LAMMPS from GitHub or their website and intend to use Kokkos for acceleration, there is an issue with Cray MPICH and `Kokkos <= 4.3`. 
+For LAMMPS to work correctly on our system, you need a LAMMPS version which provides `Kokkos >= 4.4`. 
+   Alternatively, the CMake variable `-DEXTERNAL_KOKKOS=yes` should force CMake to use the Kokkos version provided by the uenv, rather than the one contained within the lammps distribution.
```

Comment:
I guess this should be indented like this:
```suggestion
    If you are downloading LAMMPS from GitHub or their website and intend to use Kokkos for acceleration, there is an issue with Cray MPICH and `Kokkos <= 4.3`. 
    For LAMMPS to work correctly on our system, you need a LAMMPS version which provides `Kokkos >= 4.4`. 
    Alternatively, the CMake variable `-DEXTERNAL_KOKKOS=yes` should force CMake to use the Kokkos version provided by the uenv, rather than the one contained within the lammps distribution.
```
? Only the first sentence is currently rendering within the warning.